blockchain/blockdao/blockdao.go

```diff
@@ -55,17 +55,28 @@ type (
 		bodyCache    cache.LRUCache
 		footerCache  cache.LRUCache
 		tipHeight    uint64
+		stopHeight   uint64 // for tools/timemachine
```

Comment:
why we need a stop height for blockdao?

Reply:
`if err := checker.CheckIndexer(ctx, indexer, 0, func(height uint64) {`   
0 means get  `tipHeight`

Reply:
I still do not get it.

Reply:
fixed.  delete stop height in blockDAO which is repeated and meaningless.